projects/crud/src/lib/components/model-form/model-form.component.html

```diff
@@ -9,7 +9,7 @@
                 <div class="header_btn_container" *ngIf="viewConfig?.metadata?.formActions && mode==='edit'">
                     <span class="header_btns">
                         <ng-container>
-                            <button mat-raised-button id="capture-btn" *ngIf="captureIdButton?.isShown" class="header_btn"
+                            <button mat-raised-button *ngIf="captureIdButton?.isShown" class="header_btn capture_ID"
                                 [ngStyle]="getStyles(captureIdButton)" id="{{captureIdButton.name}}-btn"
```

Comment:
Because I found another id here, I believe that this is the id used for the automation. The element can't have more than one id so I removed the other one.